unattended_scripts/wazuh_install.sh

```diff
@@ -288,24 +288,17 @@ function main() {
         checkNames
     fi
 
+    installPrerequisites
```

Comment:
Why do we need to install all the prerequisites even if we only want to create the certificates?

Reply:
In order to install OpenSSL previously, there are distros that are not installed and cannot generate the certificates.

Reply:
The `installPrerequisites` function installs more utilities than `openssl`. If we use the `wazuh-cert-tool.sh`, this tool checks if the OpenSSL is installed for certificate generation and, if not, it will exit with an error. Because we are not installing anything, but creating certificates. 

So, we are changing this behavior with this change. In a host where we only want to create certificates, `tar`, `gnugp`,  an a lot of more software will be installed unnecessarily. So, `installPrerequisites` must be used in AIO, Elasticsearch, Wazuh, and Kibana cases, not with creating certificates. In the case of AIO using a Linux distro without OpenSSL (for the automatic certification creation), we can use that OpenSSL is needed by wazuh, so the `installPrerequisites` call must be done before to create the certificates.

Reply:
Done